src/Aspire.Dashboard/Resources/ControlsStrings.resx

```diff
@@ -386,4 +386,10 @@
   <data name="StructuredFilteringAddFilter" xml:space="preserve">
     <value>Add filter</value>
   </data>
+  <data name="Empty" xml:space="preserve">
+    <value>(Empty)</value>
+  </data>
+  <data name="Unset" xml:space="preserve">
+    <value>(Unset)</value>
+  </data>
```

Comment:
The resource names are not very specific. Something like `Empty/UnsetDimensionValueLabel` would be clearer.

Adding `<comment>` elements would help the loc team disambiguate between possible translations.

Reply:
These names were consistent with names for (None) and (All). I renamed them all to `LabelNone`, `LabelEmpty`, etc.